app/products/calls/components/call_notification/call_notification.tsx

```diff
@@ -0,0 +1,227 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useEffect} from 'react';
+import {useIntl} from 'react-intl';
+import {Pressable, StyleSheet, Text, View} from 'react-native';
+
+import {switchToChannelById} from '@actions/remote/channel';
+import {fetchProfilesInChannel} from '@actions/remote/user';
+import {dismissIncomingCall} from '@calls/actions/calls';
+import {leaveAndJoinWithAlert} from '@calls/alerts';
+import CallAvatar from '@calls/components/call_avatar';
+import {removeIncomingCall} from '@calls/state';
+import {ChannelType, type IncomingCallNotification} from '@calls/types/calls';
+import CompassIcon from '@components/compass_icon';
+import FormattedText from '@components/formatted_text';
+import {Preferences} from '@constants';
+import {CALL_NOTIFICATION_BAR_HEIGHT} from '@constants/view';
+import {useServerUrl} from '@context/server';
+import DatabaseManager from '@database/manager';
+import WebsocketManager from '@managers/websocket_manager';
+import ChannelMembershipModel from '@typings/database/models/servers/channel_membership';
+import {changeOpacity} from '@utils/theme';
+import {typography} from '@utils/typography';
+import {displayUsername} from '@utils/user';
+
+const style = StyleSheet.create({
+    outerContainer: {
+        backgroundColor: Preferences.THEMES.denim.onlineIndicator,
```

Comment:
why only the denim theme?

Reply:
Because it's not a themed design -- I'm using those as named constants instead of the hex values. I can use hex values instead if you prefer?